src/styles/common/overwrite.styl

```diff
@@ -26,7 +26,7 @@
     padding: 3px 5px 0
     min-height: 29px
     height: 29px
-    overflow: scroll
+    overflow-y: auto
```

Comment:
What about `overflow-x`? I I do have both scrolls 

![image](https://user-images.githubusercontent.com/166626/129492917-6c89620c-178e-4dff-9b13-82a2250c05e4.png)

Reply:
when nothing specified for overflow it means that it will use default value (we need to alter only vertical scroll behavior)